WordPress/src/main/java/org/wordpress/android/viewmodel/domains/DomainRegistrationDetailsViewModel.kt

```diff
@@ -213,6 +215,7 @@ class DomainRegistrationDetailsViewModel @Inject constructor(
     @Subscribe(threadMode = ThreadMode.MAIN)
     fun onCartRedeemed(event: OnShoppingCartRedeemed) {
         if (event.isError) {
+            AnalyticsTracker.track(Stat.AUTOMATED_TRANSFER_CUSTOM_DOMAIN_PURCHASE_FAILED)
```

Comment:
There is an injectable `AnalyticsTrackerWrapper` class which I think should be used in testable classes like `ViewModels`. This way you can add a check into your unit test that the event actually gets fired. It's ok to use the static methods in Fragments and Activities 👍